experiment/autobumper/bumper/bumper.go

```diff
@@ -94,25 +82,32 @@ func (af *fileArrayFlag) Set(value string) error {
 
 // Options is the options for autobumper operations.
 type Options struct {
```

Comment:
Some of the options are optional, or has some other custom restrictions as defined in `validateOptions` function below. So you may want to add `omitempty` on some of the fields

Reply:
It seems that any entry that is not in the yaml is automatically set to the default value even without omitempty so omitempty should not be necessary.

Reply:
ack